BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Networking/GiniBank+Networking.swift

```diff
@@ -82,24 +82,63 @@ extension GiniBank {
+     - Parameter trackingDelegate: A delegate object to receive user events
+
+     - Returns: A presentable view controller.
+     */
+    public class func viewController(withClient client: Client,
+                                     importedDocuments: [GiniCaptureDocument]? = nil,
+                                     configuration: GiniBankConfiguration,
+                                     resultsDelegate: GiniCaptureResultsDelegate,
+                                     pinningConfig: [String: [String]],
+                                     documentMetadata: Document.Metadata? = nil,
+                                     api: APIDomain = .default,
+                                     userApi: UserDomain = .default,
+                                     trackingDelegate: GiniCaptureTrackingDelegate? = nil) -> UIViewController {
+        let screenCoordinator = GiniBankNetworkingScreenApiCoordinator(client: client,
+                                                                       resultsDelegate: resultsDelegate,
+                                                                       configuration: configuration,
+                                                                       pinningConfig: pinningConfig,
+                                                                       documentMetadata: documentMetadata,
+                                                                       api: api,
+                                                                       userApi: userApi,
+                                                                       trackingDelegate: trackingDelegate)
+        return screenCoordinator.startSDK(withDocuments: importedDocuments)
+    }
+
     // MARK: - Screen API with Custom Networking - Initializers for 'UIViewController'
 
     /**
      Returns a view controller which will handle the analysis process.
      It's the easiest way to get started with the Gini Bank SDK as it comes pre-configured and handles
      all screens and transitions out of the box with the custom networking.
      
-     - parameter importedDocuments: There should be either images or one PDF, and they should be validated before calling this method.
-     - parameter resultsDelegate: Results delegate object where you can get the results of the analysis.
-     - parameter configuration: The gini bank configuration to set.
-     - parameter documentMetadata: Additional HTTP headers to send when uploading documents.
-     - parameter trackingDelegate: A delegate object to receive user events.
-     - parameter networkingService: A delegate object which implement protocol for the document processing events.
-     - parameter configurationService: A delegate object which implement protocol for the customer configuration.
+     - Parameter importedDocuments: There should be either images or one PDF, and they should be validated before calling this method.
```

Comment:
[Suggestion]: I think we can remove Parameter from each lines, you already doing it below, in Bank SDK